storm-core/test/clj/org/apache/storm/nimbus_test.clj

```diff
@@ -30,6 +30,7 @@
   (:import [java.util HashMap])
   (:import [java.io File])
   (:import [org.apache.storm.utils Time Utils])
+  (:import [org.apache.storm.zookeeper Zookeeper])
```

Comment:
yes， I don't know how to be better to port zk-leader-elector by java at nimbus_test.clj